module_fcst_grid_comp.F90

```diff
@@ -745,19 +744,8 @@ subroutine fcst_initialize(fcst_comp, importState, exportState, clock, rc)
     endif
 ! if to write out restart at the end of forecast
     restart_endfcst = .false.
-    if ( ANY(frestart(:) == total_inttime) ) restart_endfcst = .true.
```

Comment:
Is `restart_endfcst` now always .false. ? If it is `if ( restart_endfcst )` block from fcst_finalize can be removed.

Reply:
Done

Reply:
Thanks. I do not know if you want to remove if(restart_endfcst) blocks in atmos_model.F90 in `atmos_model_end`, in which case restart_endfcst logical variable can be removed from module_fv3_config. This can be cleaned up later.